.github/workflows/wheels-Windows-MacOS.yml

```
@@ -0,0 +1,55 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Wheels for Windows and MacOS
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-2016, macOS-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8] 
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade -q pip pytest wheel setuptools twine
+    - name: Test with pytest
+      run: |
+        python setup.py install
```

Running `python setup.py foo` is discouraged, instead:
```suggestion
        pip install .
```